docs/tutorials/upgrade-a-chain/sudo-upgrade.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Use the Sudo Pallet
+---
+
+As the name of the [Sudo pallet](../../knowledgebase/runtime/frame#sudo) implies, it provides
+capabilities related to the management of a single
+[`sudo` ("superuser do")](https://en.wikipedia.org/wiki/Sudo) administrator. In FRAME, the `Root`
+Origin is used to identify the runtime administrator; some of FRAME's features, including the
+ability to update the runtime by way of
+[the `set_code` function](https://substrate.dev/rustdocs/v2.0.0-rc6/frame_system/enum.Call.html#variant.set_code),
+are only accessible to this administrator. The Sudo pallet maintains a single
+[storage item](../../knowledgebase/runtime/storage): the ID of the account that has access to the
+pallet's [dispatchable functions](../../knowledgebase/getting-started/glossary#dispatch). The Sudo
+pallet's `sudo` function allows the holder of this account to invoke a dispatchable as the `Root`
+origin. The following pseudo-code demonstrates how this is achieved, refer to the
+[Sudo pallet's source code](https://github.com/paritytech/substrate/blob/v2.0.0-rc6/frame/sudo/src/lib.rs)
+to learn more.
+
+```rust
+fn sudo(origin, call) -> Result {
+  // Ensure caller is the account identified by the administrator key
+  let sender = ensure_signed(origin)?;
+  ensure!(sender == Self::key(), Error::RequireSudo);
+
+  // Dispatch the specified call as the Root origin
+  let res = call.dispatch(Origin::Root);
+  Ok()
+}
+```
+
+## Start the Template Node
+
+Since forkless runtime upgrades do not require network participants to restart their blockchain
+clients, the first step of this tutorial is to start the template node as-is. Build and start the
```

Comment:
I would also a a note here saying that they should leave this node running throughout the tutorial since this is what shows the forkless nature of forkless upgrades